packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol

```diff
@@ -391,4 +391,48 @@ contract L1StandardBridge_Test is Bridge_Initializer {
             hex""
         );
     }
+
+    function test_finalizeBridgeERC20FailSendBack() external {
+        deal(address(BadL1Token), address(L1Bridge), 100, true);
+
+        uint256 slot = stdstore
+            .target(address(L1Bridge))
+            .sig("deposits(address,address)")
+            .with_key(address(BadL1Token))
+            .with_key(address(L2Token))
+            .find();
+
+        // Give the L1 bridge some ERC20 tokens
+        vm.store(address(L1Bridge), bytes32(slot), bytes32(uint256(100)));
+        assertEq(L1Bridge.deposits(address(BadL1Token), address(L2Token)), 100);
+
+        vm.expectEmit(true, true, true, true);
+
+        emit ERC20BridgeInitiated(
+            address(BadL1Token),
+            address(L2Token),
+            alice,
+            alice,
```

Comment:
This misses the possibility (which we've gotten wrong previously), that the tokens are returned to the wrong sender on the other domain. I'd suggest this change and the one below:
```suggestion
            bob,
            alice,
```

Reply:
Done

Reply:
jc that's fast